crates/python/Cargo.toml

```diff
@@ -10,7 +10,7 @@ categories = ["api-bindings", "compilers", "science", "emulators"]
 readme = "./README.md"
 
 [features]
-libquil = ["qcs/libquil"]
+# libquil = ["qcs/libquil"]
```

Comment:
?

Reply:
Ahh right, I did this to run tests locally on my MBP with getting libquil errors. Would be nice to support `cargo make dev-flow` without libquil.

Reply:
Since the dependency is optional, I assume it's intentional that `dev-flow` turns it on. I don't know for sure, though.

Reply:
Oh - yup looks like `dev-test-flow` is a cargo make predefined task which tests with `all-features`. So locally I need to `cargo make pytest-flow && cargo make stubtest`.